ppp/iteration_2_24102022.md

```diff
@@ -0,0 +1,23 @@
+### Plans 📆
+
+- Reiterate on the wireframes and polish them. (Hoping for feedback from the community 🤞)
```

Comment:
do you mean 'create full-fledged mock-ups' by "polishing them"?

Reply:
do we need to be specific? 🙈 I meant getting it to a point where we feel comfortable implementing it.